### PR TITLE
New Website Design!

### PR DESCRIPTION
**Pull request by amarlearning**
Changes made in this PR.

1. New Website design.
2. Removed Eventbrite extension.
3. Change coding style (Tabs instead of spaces)

_[Screenshot]_
screenshot from 2017-06-19 06-43-58
screenshot from 2017-06-19 06-44-16
screenshot from 2017-06-19 06-44-28

### CODE REVIEW

File: oshc/main/templates/index.html [hunk 1]
```diff
@@ -1,60 +1,106 @@
...
+							<h3 class="panel-title">OSC Chat Session with Himanshu Mishra</h3>
+						</div>
+						<div class="panel-body">
+							<a href="https://orkohunter.net">Himanshu</a> is a pre-final year undergraduate student at IIT Kharagpur. He has been a Google Summer of Code student in 2015 and 2016 and is currently a mentor in Python Software Foundation.
+							<div class="line-gap-10px"></div>
+							<i>Date: Wed, June 21, 2017 8:30 PM – 9:30 PM IST</i>
+							<div class="line-gap-10px"></div>
+							<button type="button" class="btn btn-lg btn-primary sRegister">
+								<a href="https://www.eventbrite.com/e/open-source-contributors-chat-session-with-himanshu-mishra-tickets-35227478336?ref=ecal">Register Now!</a>
+							</button>
```
Comment (tapaswenipathak) [inline oshc/main/templates/index.html:47]: @amarlearning The new design looks great! 👍 But after this change one has to go and edit the code for new sessions and also remove the previous ones, isn't it? This adds to extra effort before every new session, do you have any ideas for this?

File: oshc/main/templates/base.html [hunk 2]
```diff
@@ -1,49 +1,40 @@
...
+				</ul>
+			</div>
+		</div>
+	</nav>
+
+	<div id="content">
+	{% block content %}{% endblock %}
+	</div>
+
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (tapaswenipathak) [inline oshc/main/templates/base.html:36]: @amarlearning Did you test it on small screens?

**Comment by tapaswenipathak** [comment]
Everyone, I want to test this once then anyone can go ahead and merge it. :)

@amarlearning We really need to think if we can remove the need to update the code before every new session. :(

File: oshc/main/templates/base.html (hunk 2 above)
Comment (amarlearning) [inline oshc/main/templates/base.html]: Built with Bootstrap so it's mostly responsive. Have to write `media-query` for some overflows. I was waiting for everyone's approval on the design. I am a little lazy :sweat_smile:

Comment (amarlearning) [reply oshc/main/templates/base.html:36]: Talking about the dropdown menu? Yeah, that works fine!

**Comment by vaibhavsingh97** [comment]
@amarlearning I can help you out for testing and fixing different screens

File: oshc/main/templates/base.html (hunk 2 above)
Comment (tapaswenipathak) [reply oshc/main/templates/base.html:36]: Great!

**Comment by amarlearning** [comment]
> But after this change one has to go and edit the code for new sessions and also remove the previous ones, isn't it? This adds to extra effort before every new session.

Exactly what I thought when I was working on this and I have two solutions for this:-

1. One way is to maintain a JSON file which contains objects of various events and we can use that JSON to generate Html code via JavaScript dynamically.

2. Another way is to create a table in Database for such events and later use that to generate pages.

_In both, the case one has to update (JSON or Database) from time to time._

I can implement both the solutions. Let me know your thoughts on this or If you have any other idea, please share.

**Comment by amarlearning** [comment]
@vaibhavsingh97 Thanks, man! 
Once I am done with this I will let know for testing :sweat_smile:

**Comment by vaibhavsingh97** [comment]
@amarlearning we can use database as I think it's easier to scale database than a directory full of files.
Also what I think that using JSON can be risky (security wise). Correct me if I am wrong.

**Comment by tapaswenipathak** [comment]
@amarlearning Let's have it in a database, anyway for other changes we really need it. 

For now let's have this. 🎆  It is much better than the Eventbrite plugin. 💐

**Pull request merged**